src/taskgraph/run-task/run-task

```diff
@@ -275,6 +275,7 @@ def run_command(prefix, args, *, extra_env=None, cwd=None):
         cwd=cwd,
         env=env,
     )
+    _display_python_version_in_sub_process(subprocess)
```

Comment:
Nit: I'm sorry, I'm not sure to follow. What's the intent about passing down `subprocess`? This module is available everywhere in this file since it's imported at the top-level: 

https://github.com/taskcluster/taskgraph/blob/4346754b8457dd53f8b541297b8d60d502c75c0c/src/taskgraph/run-task/run-task#L37

Reply:
ok, I will update